src/composer.ts

```diff
@@ -627,8 +634,24 @@ export class AtomicTransactionComposer {
     }
 
     const stxns = await this.gatherSignatures();
+    const txnObjects: EncodedSignedTransaction[] = stxns.map(
+      (stxn) => encoding.decode(stxn) as EncodedSignedTransaction
+    );
+
+    const currentRequest: SimulateRequest = new SimulateRequest({
+      txnGroups: [
+        new SimulateRequestTransactionGroup({
+          txns: txnObjects,
+        }),
+      ],
+      allowMoreLogging: request == null ? false : request.allowMoreLogging,
+      allowEmptySignatures:
+        request == null ? false : request.allowEmptySignatures,
```

Comment:
Since we know we want to add more properties to `SimulateRequest` in the future and it is a generated class here, it would be great to avoid having to manually reference all the properties here

Reply:
That looks good!